test/pom.xml

```diff
@@ -306,6 +306,12 @@ THE SOFTWARE.
         <skipTests>true</skipTests>
       </properties>
     </profile>
+    <profile>
+      <id>smoke-test</id>
+      <properties>
+        <test>jenkins.model.StartupTest,jenkins.model.JenkinsTest,jenkins.install.*,jenkins.security.CustomClassFilterTest,hudson.AboutJenkinsTest,hudson.ClassicPluginStrategyTest,hudson.LauncherTest,hudson.slaves.JNLPLauncherTest,hudson.model.RunTest,hudson.model.UserTest,hudson.model.FreeStyleProjectTest,hudson.model.HudsonTest,hudson.model.ComputerTest,hudson.CLI.BuildCommandTest#sync,*.smokes</test>
```

Comment:
Will appreciate suggestions. IMHO it could be ported to the master branch for local tests